doc/api/module.md

```diff
@@ -534,14 +607,15 @@ provided, all `require` calls from this module will be processed by the ESM
 loader with registered `resolve` and `load` hooks; all `require.resolve` calls
 from this module will be processed by the ESM loader with registered `resolve`
 hooks; `require.extensions` and monkey-patching on the CommonJS module loader
-will not apply. If `source` is undefined or `null`, it will be handled by the
-CommonJS module loader and `require`/`require.resolve` calls will not go through
-the registered hooks. This behavior for nullish `source` is temporary — in the
-future, nullish `source` will not be supported.
-
-The Node.js own `load` implementation, which is the value of `next` for the last
-loader in the `load` chain, returns `null` for `source` when `format` is
-`'commonjs'` for backward compatibility. Here is an example loader that would
+will not apply. (In other words, handled `require` calls will behave similarly
+to an `import` of a CommonJS module.) If `source` is undefined or `null`, it
```

Comment:
> handled `require` calls will behave similarly to an `import` of a CommonJS module

Well not exactly, one is async, the other one is sync, that's quite a difference.